### DISCUSSION TITLE
backbon swin large to backbon swin tiny

### DISCUSSION THREAD

**Discussion by aws-ahmad** [yoshitomo-matsubara/torchdistill#311] [Q&A] [answered]
Hi
Now i'm trying to distillate pretrained model backbon swin large to swin tiny.
I have both model as .pth file and custom dataset
please tell me how to import my ready models .pth

**Comment by yoshitomo-matsubara** [comment, marked as answer]
Hi @aws-ahmad 

Some people have already asked similar questions e.g., https://github.com/yoshitomo-matsubara/torchdistill/discussions/264
Please choose `Q&A` category when opening discussions if the questions are not open-ended so that you can close this discussion by marking this as answer.

**Reply by aws-ahmad** [reply]
Thank you
I will check #264
